### DISCUSSION TITLE
Can I deploy a R web app with heroku's buildpack?

### DISCUSSION THREAD

**Discussion by F041** [dokku/dokku#6881] [General]
I didn't quite understand if I can use heroku's R buildpack, as done here: 
https://www.r-bloggers.com/2018/12/running-an-r-script-on-heroku/
Using what indicated here: https://dokku.com/docs/deployment/builders/herokuish-buildpacks/

**Comment by josegonzalez** [comment]
Yes you can, but you probably want a newer version of the buildpack mentioned in that post.

```
dokku buildpacks:set $APP https://github.com/virtualstaticvoid/heroku-buildpack-r.git#main
```

That should work,
